scripts/plot_dataframe.py

```diff
@@ -35,55 +32,113 @@
 logger = logging.getLogger(__name__)
 _CONF: Dict[str, Any] = {}
 
+
+# Update the global variable TO_DISPLAY to select which indicator you want to display
+TO_DISPLAY = {
```

Comment:
honestly, i don't like the fact that the script needs to be changed to change the displayed indicators.

would something like `--indicators1 sma,ema20,ema40 --indicators2 macd,rsi` specifying the dataframe's non-default columns make sense? (with sensible defaults of course)

Rest looks good to me

Reply:
Done. Now you can use `--indicators1 sma,ema20,ema40 --indicators2 macd,rsi`